chef-gpt.py

```diff
@@ -21,20 +25,25 @@
 messages.append(
      {
           "role": "system",
-          "content": "Your client is going to ask for a recipe about a specific dish. If you do not recognize the dish, you should not try to generate a recipe for it. Do not answer a recipe if you do not understand the name of the dish. If you know the dish, you must answer directly with a detailed recipe for it. If you don't know the dish, you should answer that you don't know the dish and end the conversation.",
+          "content": "Your client is going to ask for a recipe about a specific dish. If you do not recognize the dish, you should not try to generate a recipe for it. Do not answer a recipe if you do not understand the name of the dish. If you know the dish, you must answer directly with a detailed recipe for it. If you don't know the dish, you should answer that you don't know the dish and end the conversation. Please be concise, answer in max 100 words",
      }
 )
 
 """
-Receive the name of the dish from the user and place it inside the messages list
-- wait for user input
-- append input to messages
+Receive the name of the dish and the preferred chef personality from the user
 """
 dish = input("Type the name of the dish you want a recipe for:\n")
+chef_personality = input("Choose your chef (1 for experienced chef, 2 for Italian pasta expert):\n")
+
+if chef_personality == "1":
+    personality = "experienced chef"
+else:
+    personality = "Italian pasta expert"
+
 messages.append(
     {
         "role": "user",
-        "content": f"Suggest me a detailed recipe and the preparation steps for making {dish}"
+        "content": f"As the {personality}, suggest me a detailed recipe and the preparation steps for making {dish}"
```

Comment:
Here we will feed in the request one of the two personalities choosen by the user.